spec/model/semester_spec.rb

```diff
@@ -3,99 +3,98 @@
 require_relative '../../app/model/semester'
 require_relative '../../app/model/base'
 require_relative '../../app/model/competency_level'
-require 'pry'
 
 describe Semester do
 
   before do
     Semester.base_path = 'spec/'
     CompetencyLevel.base_path = 'spec/'
   end
-  
+
   let(:semesters) { Semester.all }
 
   it 'should confirm level in semester 1' do
     semester1 = Semester.find(id: 1)
     semester2 = Semester.find(id: 2)
-    
+
     in_semester1 = semester1.includes?(level: CompetencyLevel.find(id: '1.1.1'))
     in_semester2 = semester2.includes?(level: CompetencyLevel.find(id: '1.1.1'))
-    
+
     expect(in_semester1).to be true
```

Comment:
expect(semester1).to include(...)

Reply:
The goal is to test the method includes? from semester which returns a boolean

Reply:
exactly ... I think rspec tries to call the method includes? if you're doing like this:

includes(..)